core/src/test/java/cucumber/runtime/formatter/HTMLFormatterTest.java

```diff
@@ -58,8 +53,8 @@ public void writes_index_html() throws Throwable {
         assertEquals("", reportElement.text());
     }
 
-    @Test
-    public void writes_valid_report_js() throws Throwable {
+    @Test(expected = EcmaError.class)
+    public void empty_report_js_throws_error() throws Throwable {
```

Comment:
Wasn't this this the test that caused the problems with the locale because the EcmaError contained a localized message?

Reply:
Yes it was. But I cannot reproduce that either by setting my laptops locale or passing a VM argument (see PR description).
I can only reproduce it by explicitly setting the Locale in the test and the resetting after.

Reply:
As discussed, removed the test that checks for error and replaced with test that verifies content of reportJs.